zowe/zowe-cli
# Generate different base profile names for global and project configs

**What It Does**


Both the 'zowe config init' and 'zowe config auto-init' commands generate a base profile name of 'global_base' for a global config file and 'project_base' for a project config file.

**How to Test**


Run the commands and check out the name of the base profile in the resulting zowe.config.json file.

**Review Checklist**
I certify that I have:


**Additional Comments**
